src/www/ui/api/documentation/openapi.yaml

```diff
@@ -550,6 +550,18 @@ paths:
         - Search
       description: Search the FOSSology for a specific file
       parameters:
+        - name: limit
+          required: false
+          schema:
+            type: integer
+            minimum: 1
+          in: header
+        - name: page
+          required: false
```

Comment:
Please add `description` for `limit` and `page`

Reply:
@GMishx updated it.